samples/aspnetcore/blazor/InputLargeTextArea/src/InputLargeTextArea/InputLargeTextArea.cs

```diff
@@ -99,7 +99,7 @@ public virtual async ValueTask<StreamReader> GetTextAsync(int maxLength = 32_000
                 // 0 length streams/textareas aren't permitted from JS->.NET Streaming Interop.
                 if (jsException.InnerException is ArgumentOutOfRangeException outOfRangeException &&
                     outOfRangeException.ActualValue is not null &&
-                    outOfRangeException.ActualValue is int actualLength &&
+                    long.TryParse(outOfRangeException.ActualValue.ToString(), out var actualLength) &&
```

Comment:
Instead of converting to string and `TryParse`'ing to a `long`, can we just do:

```
outOfRangeException.ActualValue is long actualLength
```

Reply:
Yeah the reason I did it that way, as stated before, I don't know if it's an int in another version of .NET. why was it being parsed as an int in the first place?

I think this is the safest way. Then we're not bound to the type. It's not like we're dealing with a high performance situation anyway 🙂

Reply:
> I don't know if it's an int in another version of .NET. why was it being parsed as an int in the first place?

I don't see why it would be an int, as far as I know [stream lengths](https://docs.microsoft.com/en-us/dotnet/api/system.io.stream.length?view=net-6.0) should always be `long`s.

Even the `JSStreamReference` uses `long`: 

https://github.com/dotnet/aspnetcore/blob/c85baf8db0c72ae8e68643029d514b2e737c9fae/src/JSInterop/Microsoft.JSInterop/src/Implementation/JSStreamReference.cs#L13-L14

https://github.com/dotnet/aspnetcore/blob/c85baf8db0c72ae8e68643029d514b2e737c9fae/src/JSInterop/Microsoft.JSInterop/src/IJSStreamReference.cs#L11-L14

`long` should be safe, and correct. This being an `int` initially was likely just an oversight on my part initially. I'm hesitant to use the `TryParse` logic if it isn't necessary, as it could just cause confusion later.

Reply:
I've updated the code to use long instead :)